### ISSUE TITLE
Question regarding fig. 2 in your paper

### ISSUE THREAD

**Issue by cityCoder** [tariks/peakachu#17]
Hello,

I have a question regarding figure 2 in your paper. The figure said the recalls are 60%~80% when using ChIA-PET or HiChIP data as target. But peakachu's prediction is around 15K, and the number of peaks in CTCF ChIA-PET is almost 100K.  Did you pool ChIA-PET as well?

**Comment by tariks** [comment]
Yes, we pooled ChIA-PET coordinates and allowed for a +/- 1 bin tolerance when determining matches.

**Comment by cityCoder** [comment]
Thanks. It makes more sense! 

How did you pool the ChIA-PET data? When I tried to use the peakachu's pooling tool to pool ChIP-PET, I got 0 loops. 

I covert ChIA-PET data to the input of pool tool as follows:
chr9	99230000	99235000	chr9	100010000	100015000
-->
chr9	99230000	99235000	chr9	100010000	100015000	1	IF 

Thanks!
